docs/UserGuide.md

```diff
@@ -233,7 +240,7 @@ Format: `delete-tag t/TAGNAME [f/]`
 * The `Tag` cannot be assigned to any contacts.
 * If the `Tag` is in used, you can use `f/` to force the deletion of the `Tag` and unassign this tag from all contacts.
 
-### Wedding Features
+## Wedding Features
 
 ### Adding a Wedding : `create-wedding`
```

Comment:
This is for line 265 since I can't add a comment there - should we give an example of how the force tag is used?

Reply:
Line 251 - maybe can change to "the weddingname should be alphanumeric" rather than "is alphanumeric"?